package.json

```diff
@@ -6,24 +6,25 @@
   "scripts": {
     "predev": "yarn codegen",
     "dev": "ts-node-dev --no-notify --respawn --transpile-only src/server",
-    "start": "node dist/server",
+    "start": "./node_modules/.bin/nodemon --optimize_for_size --max_old_space_size=920 --gc_interval=100 dist/server.js",
```

Comment:
Nodemon to restart the thing if it crashes. Also node flags to tell V8 to GC early and often instead of acting like Chrome and consuming like 1.5 GB before cleaning up after itself.